src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -379,8 +379,8 @@ function packScalarsToVector(
 }
 
 /** @returns a set of flushed and non-flushed floating point results for a given number. */
-function calculateFlushedResults(value: number): Set<Scalar> {
-  return new Set([f64(value), f64(flushSubnormalNumber(value))]);
+function calculateFlushedResults(value: number): Array<Scalar> {
+  return new Array<Scalar>(f64(value), f64(flushSubnormalNumber(value)));
```

Comment:
I think you can just use the short form `[ f64(value), f64(flushSubnormalNumber(value)) ]` here

Reply:
Done.